tests/AutoReview/FixerFactoryTest.php

```diff
@@ -183,7 +183,6 @@ public function provideFixersPriorityCases()
             [$fixers['phpdoc_order'], $fixers['phpdoc_trim']],
             [$fixers['phpdoc_return_self_reference'], $fixers['no_superfluous_phpdoc_tags']],
             [$fixers['phpdoc_scalar'], $fixers['phpdoc_to_return_type']],
-            [$fixers['phpdoc_separation'], $fixers['phpdoc_trim']],
```

Comment:
When deleting this, you need to also delete this pair from the `$casesWithoutTests` array in `testFixersPriorityPairsHaveIntegrationTest` (in this class).

Reply:
:+1:  thanks, updated